src/main/java/com/checkmarx/ast/wrapper/CxConstants.java

```diff
@@ -3,14 +3,14 @@
 public final class CxConstants {
 
     public static final String SOURCE = "-s";
-    public static final String VERBOSE = "-v";
     public static final String PROJECT_NAME = "--project-name";
-    public static final String SCAN_TYPES = "--scan-types";
```

Comment:
@AndreGCX  I think for scan create is scan types, because you can have more than one, so we can't change this.

Reply:
It was added it back